src/FlatButton/FlatButton.js

```diff
@@ -293,6 +298,7 @@ class FlatButton extends Component {
         onMouseEnter={this.handleMouseEnter}
         onTouchStart={this.handleTouchStart}
         style={mergedRootStyles}
+        disableTouchRipple={disableTouchRipple}
```

Comment:
No need to explicit the property, it's going to be applied on the right element.